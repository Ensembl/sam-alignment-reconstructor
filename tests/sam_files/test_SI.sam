MS2007487-600V3:1:2104:28968:11326	147	cecr1b_T2_4_5106979-5107305	39	70	1S110M3I179M	=	1	-326	AAAGGCTCTTTCAAACCCTGGTAGATGTGATCCTTCAGGACAGGAGCATATGTAATCAGCCCAGACAGAGCAATGAAAATCTTCTCAAACCGCTTCCATATATAGTCCTATATAATAGGATAAGACGTCTCCGGGTCTTCCGTAAAAAGCGTTAGCTTACGCATCAAGCTGAAAGATAATGGCTGAGTTAGTCTACAGTAGTCAACGTTTGGATTTGGTTAATTAAATGAGAGCTATTATGCCAAACTCACTCTTTAACTCACTTGTTGTTTAAAAAAGGTTGTGTGGCAACA	,?++?C89C?+**<37CGFC>6A614A84683*)CBC9)CAB=C589*CE>7+?A;*C9*F6FFFC6A6=*?4+CC;@FFA+>AB5AFFGGCC6E?@+8+720GFDD=,GFFFGGFFB,GGFGGGGE6CECGE?BFCGGE>AGGCGGDC8DFF9,,@GGGFGD:GGGDGGGGGGECFFFA9,,,GGGFAGFGFFGGGGGGGGE?GGFF,FCGFGGGGFCEFF<GFFGGFDFFFGGGGGFFE8,FFFCFGGGGGGGGGGGGGGDGFCGGGGGGGGFGFDEGGGEEGCGCGGGGF	PG:Z:novoalign	AS:i:260	UQ:i:260	NM:i:10	MD:Z:2A49G55G1G1G38A86A50	PQ:i:466	SM:i:0	AM:i:0	BC:Z:TCAAGA
