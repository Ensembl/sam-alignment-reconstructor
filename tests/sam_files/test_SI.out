MS2007487-600V3:1:2104:28968:11326
.AAAGCTCTTTCAAACCCTGGTAGATGTGATCCTTCAGGACAGGAGCATA
 ||:||||||||||||||||||||||||||||||||||||||||||||||
aAAGGCTCTTTCAAACCCTGGTAGATGTGATCCTTCAGGACAGGAGCATA

TGTGATCAGCCCAGACAGAGCAATGAAAATCTTCTCAAACCGCTTCCATA
|||:||||||||||||||||||||||||||||||||||||||||||||||
TGTAATCAGCCCAGACAGAGCAATGAAAATCTTCTCAAACCGCTTCCATA

TATAGTCCTGT---GTGGGATAAGACGTCTCCGGGTCTTCCGTAAAAAGC
|||||||||:|   :|:|||||||||||||||||||||||||||||||||
TATAGTCCTATATAATAGGATAAGACGTCTCCGGGTCTTCCGTAAAAAGC

GTTAGATTACGCATCAAGCTGAAAGATAATGGCTGAGTTAGTCTACAGTA
|||||:||||||||||||||||||||||||||||||||||||||||||||
GTTAGCTTACGCATCAAGCTGAAAGATAATGGCTGAGTTAGTCTACAGTA

GTCAACGTTTGGATTTGGTTAATTAAATGAGAGCTATTATGCAAAACTCA
||||||||||||||||||||||||||||||||||||||||||:|||||||
GTCAACGTTTGGATTTGGTTAATTAAATGAGAGCTATTATGCCAAACTCA

CTCTTTAACTCACTTGTTGTTTAAAAAAGGTTGTGTGGCAACA
|||||||||||||||||||||||||||||||||||||||||||
CTCTTTAACTCACTTGTTGTTTAAAAAAGGTTGTGTGGCAACA

